addons/base_import/static/src/js/import.js

```diff
@@ -206,10 +206,14 @@ openerp.base_import = function (instance) {
 
         //- File & settings change section
         onfile_loaded: function () {
+            if (!(this.$('input.oe_import_file').val().split('.').pop() == "csv")) {
+                this.$el.find('.oe_import_toggled').children().nextAll().hide()
```

Comment:
* Use `this.$()`
* `.children().slice(1)` is a much better and clearer way to select all-but-the-first children of the reference node